ballerina/tests/test.bal

```diff
@@ -0,0 +1,159 @@
+}
+isolated function testPostLookup() returns error? {
+    Get2TweetsIdResponse response = check twitter->/tweets/["1806286701704462623"]();
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function testBookmarkPost() returns error? {
+    BookmarkMutationResponse response = check twitter->/users/[userId]/bookmarks.post(
+        payload = {tweet_id: "1806286701704462623"}
+    );
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function testBookmarkDelete() returns error? {
+    BookmarkMutationResponse response = check twitter->/users/[userId]/bookmarks/["1806286701704462623"].delete();
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function testRetweet() returns error? {
+    UsersRetweetsCreateResponse response = check twitter->/users/[userId]/retweets.post(
+        payload = {tweet_id: "1806286701704462623"}
+    );
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function testDeleteRetweet() returns error? {
+    UsersRetweetsDeleteResponse response = check twitter->/users/[userId]/retweets/["1806286701704462623"].delete();
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function testFollowSpecificUser() returns error? {
+    UsersFollowingCreateResponse response = check twitter->/users/[userId]/following.post(
+        payload={
+            target_user_id:"1803011651249278976"
+        }
+    );
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function testUnfollowSpecificUser() returns error? {
+    UsersFollowingDeleteResponse response = check twitter->/users/[userId]/following/["1803011651249278976"].delete();
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function muteSpecificUser() returns error? {
+    MuteUserMutationResponse response = check twitter->/users/[userId]/muting.post(
+        payload={
+            target_user_id:"1803011651249278976"
+        }
+    );
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function unmuteSpecificUser() returns error? {
+    MuteUserMutationResponse response = check twitter->/users/[userId]/muting/["1803011651249278976"].delete();
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
+
+@test:Config {
+}
+isolated function findSpecificUser() returns error? {
+    Get2UsersResponse response = check twitter->/users(ids = ["1803011651249278976"]);
+    test:assertTrue(response?.data !is ());
+    test:assertTrue(response?.errors is  ());
+}
```

Comment:
Missing newline

Reply:
Solved in: [Update mock_service and test ballerina files ](https://github.com/ballerina-platform/module-ballerinax-twitter/pull/112/commits/22f461d825fe3e85dbd202c29aa16895feb7f68e)